Katello/katello
# Fixes #30533 - Include packages with no errata

To test:
1. Create a CV with a yum repo with some errata
2. Create a package filter with some rule and check the include packages without errata checkbox for the filter.
3. Publish and verify that all packages without errata are copied over.
